js/listeners.js

```diff
@@ -101,6 +129,34 @@ addEventListener("mousedown", () => {
       },
     })
   );
+  if (
```

Comment:
Duplicated code

Reply:
same code from line 65?

Reply:
That was for spacebar. This is for mouse.

Reply:
Much better with the `createMultiShotProjectile`, But still we need abstraction something like

```
function executeMultiShot() {
    audio.shoot.play();
    projectiles.push(createMultiShotProjectile(-3), createMultiShotProjectile(3));
}
 ```